.gitignore

```diff
@@ -141,3 +141,4 @@ Ankh.NoLoad
 
 src/Hive.IO/packages/
 .vscode/
+src/Hive.IO/Hive.IO/Resources/EnergySystems/*.psd
```

Comment:
why put psd files into gitignore? Let's keep them in the repo

Reply:
are the images used open access? We should be careful with copyright

Reply:
I will redo the images.